### PR TITLE
Status message

### PR DESCRIPTION
**Pull request by baumanta**
Make the local planner output a status message. This message is converted in mavros in a HEARTBEAT. soon there will be a PR merged on the Firmware side, which will output warnings over QGC when the avoidance system does not respond or reports timeouts

### CODE REVIEW

**Comment by baumanta** [comment]
https://github.com/PX4/Firmware/pull/10933

File: local_planner/src/nodes/local_planner_node.cpp [hunk 1]
```diff
@@ -938,6 +940,7 @@ int main(int argc, char** argv) {
   bool landing = false;
   avoidanceOutput planner_output;
   bool startup = true;
+  Node.status_msg_.state = 1; //COMPANION_PROCESS_STATE_STARTING
```
Comment (jkflying) [inline local_planner/src/nodes/local_planner_node.cpp]: As soon as these are updated to the real enums I am happy with this.

Comment (baumanta) [reply local_planner/src/nodes/local_planner_node.cpp]: you are right this is ugly. The enum is already fixed and we know what it looks like. But it is defined in mavlink, therefore I now defined the same enum also in the local_planner such that we can use the same terms

File: local_planner/src/nodes/local_planner_node.h [hunk 2]
```diff
@@ -47,6 +47,18 @@ struct cameraData {
   bool received_;
 };
 
+enum class MAV_STATE {
+  MAV_STATE_UNINIT,
```
Comment (jkflying) [inline local_planner/src/nodes/local_planner_node.h]: Ok, if these need to match an external number, I would initialize it with 0, or  even number them all. Also just add a little comment above the enum declaration here to say where this comes from.

**Pull request merged**